card/src/main/java/com/adyen/checkout/card/CardComponent.kt

```diff
@@ -60,7 +60,7 @@ class CardComponent internal constructor(
         componentEventHandler.initialize(viewModelScope)
     }
 
-    internal fun observe(
+    fun observe(
```

Comment:
We should annotate these two methods with `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`